mrp_bom_note/README.rst

```diff
@@ -0,0 +1,8 @@
+MRP - BoM notes
+===============
+
+This module performs the following:
+
+1.- Creates the new field NOTES in MRP BoM list.
+
+2.- Creates the new field NOTES in MRP production, of type related with MRP BoM List.
```

Comment:
You're right, I've got to ask. Maybe is something that got lost between diferent versions, thanks for the point.